src/agent/coverage/src/source.rs

```diff
@@ -104,13 +104,18 @@ pub fn binary_to_source_coverage(
                         };
 
                         if let Some(file) = location.file() {
+                            let file_path = if cfg!(windows) {
+                                // Windows paths are case insensitive.
+                                FilePath::new(file.full_path().to_ascii_lowercase())?
+                            } else {
+                                FilePath::new(file.full_path())?
+                            };
```

Comment:
We should do this once up-front rather than every time we examine a location

Reply:
I used the `Transformer` available in the symbolic-symcache crate so we only do the modification once.